services/orchestrator/functions/requestSyncDeployment/handler.integration-test.ts

```diff
@@ -0,0 +1,63 @@
+import { DocumentClient } from 'aws-sdk/clients/dynamodb';
+import fetch from 'node-fetch';
+import { ulid } from 'ulid';
+import { expect } from 'vitest';
+
+import {
+  buildServiceEventEntity,
+  getServiceEventPK,
+  getServiceEventSK,
+} from '../../sideEffects/dynamodb/serviceEventEntity';
+import { TEST_ENV_VARS } from '../../testEnvVars';
+const documentClient = new DocumentClient();
+const ServiceEventEntity = buildServiceEventEntity(
+  documentClient,
+  TEST_ENV_VARS.TABLE_NAME,
+);
+
+const serviceId = 'serviceId';
+const applicationId = `application${ulid()}`;
+
+describe('requestSyncDeployment', () => {
+  it('should put a service event entity inside the orchestrator table', async () => {
+    const response = await fetch(
+      `${TEST_ENV_VARS.API_URL}/request-sync-deployment`,
+      {
+        method: 'post',
+        body: JSON.stringify({
+          serviceId,
+          applicationId,
+        }),
+      },
+    );
+    expect(response.status).toBe(200);
+    const body = await response.json();
+    expect(body).toEqual({
+      status: 'ACCEPTED',
+      message: 'processing',
+    });
+    const { Items: serviceEvents } = await ServiceEventEntity.query(
+      getServiceEventPK({ serviceId, applicationId }),
+    );
+    expect(serviceEvents).toContainEqual({
+      applicationId,
+      created: expect.any(String) as string,
+      entity: 'ServiceEvent',
+      eventId: expect.any(String) as string,
+      modified: expect.any(String) as string,
+      serviceId,
+    });
+
+    const serviceEvent = serviceEvents?.find(
+      ({ serviceId: sId }) => sId === serviceId,
+    );
+    if (!serviceEvent) {
+      throw new Error('must exist according to the expect above');
+    }
+    // Cleanup
+    await ServiceEventEntity.delete({
+      pk: getServiceEventPK(serviceEvent),
+      sk: getServiceEventSK(serviceEvent),
+    });
```

Comment:
It should be run regardless in an `afterAll` or `afterEach`, so the cleanup is still done if there is an error.